README.md

```diff
@@ -9,6 +9,8 @@
 - Support most popular **Linux distributions**
 - **Continuous integration tests**
 
+To create a cluster in vagrant simply run `vagrant up`
+
 For an easy way to use it, check out [**kargo-cli**](https://github.com/kubespray/kargo-cli) </br>
```

Comment:
I think vagrant is great for tests purposes, right ?
So i propose something like
```
For an easy way to use it, check out [**kargo-cli**](https://github.com/kubespray/kargo-cli) </br>
For test purposes you can either use **vagrant** by simply running `vagrant up`
```
What do you think about that ?